01-JS-DOM-APIs/Practice1-10/CSS/style.css

```diff
@@ -0,0 +1,40 @@
+* {
+  background-color: #303030;
+  color: #E8E8E8;
+  font-size: 46px;
+  font-family: cursive;
+  text-align: center;
+}
+
+p {
+  background-color: #505050 ;
+  border-radius: 0.3em;
+}
+
+.hidden {
+  opacity: 0;
+  text-align:center;
+}
+
+.button {
+  width: 15%;
```

Comment:
**minor:** try to order your css rules alphabetically to make them more readable ;)